www/src/slices/mapper/LeafletMap.tsx

```diff
@@ -31,15 +32,17 @@ let frameworkBoundary: L.GeoJSON
 
 export let LeafletMap = () => {
 
+  enableMapSet();
```

Comment:
You'll have to show me what this is used for at some point!

Reply:
I've got a collection stored in the state, of the habitats per framework
```frameworkHabitats: new Map<Framework, Habitat[]>()```
the Map bit doesn't work unless you run this command

Reply:
Ahh okay!